README.md

```diff
@@ -0,0 +1,23 @@
+# trade-tariff-identity
+
+Ruby app providing interface to Cognito allowing users to be authenticated
+
+## Getting started
+
+### AWS Locally
+
+Add the following env variables to an `.env.development.local` file:
+
+```
+AWS_PROFILE
+COGNITO_USER_POOL_ID
+COGNITO_CLIENT_ID
+```
+
+login to the aws profile with
+
+``` sh
+aws sso login --profile [profile name]
```

Comment:
Assuming roles like this is actually harder than just using ephemeral env vars copy/pasta'd from the start page IMO